filter.go

```diff
@@ -26,3 +26,65 @@ func extractMetrics(re *regexp.Regexp, in <-chan []byte, out chan<- []byte) {
 
 	close(out)
 }
+
+func SkipLuhnMatches(in <-chan []byte) <-chan []byte {
+	out := make(chan []byte)
+
+	re := regexp.MustCompile(`\D+`)
+	go skipLuhnMatches(re, in, out)
```

Comment:
Why pass that regex as an argument since it will always be the same value?
How about having it as a constant?